### PR TITLE
Add zoom option to make OSD big in small screens

### PR DESCRIPTION
**Pull request by McGiverGim**
Alternative to https://github.com/betaflight/betaflight-configurator/pull/3241

This PR fits the OSD preview into the screen for small devices. The problem is that is not usable to reorganice elements because the small size, specially in OSD HD format. But if the user needs to make it bigger, it adds a "zoom" check to make it usable. It overflows, so the user needs to do a scroll, but is possible to reorganize elements.

Zoom disabled (default):
image

Zoom enabled:
image

### CODE REVIEW

**Comment by haslinghuis** [comment]
Would love to see the zoom button for desktop while viewpoint is above 1455px :)

On Nokia G60 the right side has some overflow past the screen.

**Comment by McGiverGim** [comment]
It has no sense to have it for over 1455px, the max-width of the OSD preview is smaller than that ;)

For Nokia G60, it does not appear like in my screenshot? My Pixel 4 has a similar resolution to yours. Maybe can you paste a screenshot?

**Comment by haslinghuis** [comment]
image

**Comment by McGiverGim** [comment]
Yes, it overflows, but only when the "zoom" option is pressed. This is the expected behaviour. It's ugly but it works.

The idea is default to false, to fit nice, but make it overflow when needed to reorganize elements. We can add real scrolls to show only the part that fits and not overflow, but I think it will not be very usable.

**Comment by haslinghuis** [comment]
After playing more with this PR I found a way to add right margin:

Adding `padding-right: 1em` on line 382 does the trick :tada: 

https://stackoverflow.com/questions/32765039/using-margin-on-flex-items

**Comment by haslinghuis** [comment]
image

**Comment by McGiverGim** [comment]
> After playing more with this PR I found a way to add right margin:
> 
> Adding `padding-right: 1em` on line 382 does the trick 🎉
> 
> https://stackoverflow.com/questions/32765039/using-margin-on-flex-items

I didn't saw this message. Now all have more sense. I will change it ;)

**Comment by McGiverGim** [comment]
Added the padding to the "zoom" style only. If not, it affects the desktop too and gives different space between left and right (left only gap, right gap and padding).

**Review by haslinghuis — APPROVED** [review]
Perfect now - zoom with analog does nothing :). But this looks good!

**Comment by McGiverGim** [comment]
It depends on the size of your phone. On mine it makes it bigger with analog too 😉

File: src/js/tabs/osd.js [hunk 1]
```diff
@@ -2985,6 +2983,14 @@ osd.initialize = function(callback) {
                     // Hide custom if not used
                     $('.osdfont-selector option[value=-1]').toggle(osdFontSelectorElement.val() === -1);
 
+                    // Zoom option for the preview only for mobile devices
+                    if (GUI.isCordova()) {
+                        $('.osd-preview-zoom-group').css({display: 'inherit'});
```
Comment (chmelevskij) [inline src/js/tabs/osd.js:2988]: Why not just using css media query for that classname instead of js?

Comment (McGiverGim) [reply src/js/tabs/osd.js:2988]: The media query was my first attempt, using resolution, but finally I wanted to restrict to Android only. Is a way in the media query to limit to Android specifically?

**Review by blckmn — APPROVED** [review]

**Pull request merged**